src/params/progression.jl

```diff
@@ -34,17 +38,38 @@ function sample_severity(rng::AbstractRNG, age::Real)
   elseif age < 80;  dist = severity_dists[5]
   else;             dist = severity_dists[6]
   end
+  idx = gender + 1 |> Int32
+  prob = hospitalization_probs[idx][age + 1]
+  if age < 0;       error("age should be non-negative")
+  elseif age < 40;  dist = Categorical([0,  1.0 - prob,  prob *(1-0.030927835051546393) ,  prob *0.030927835051546393])
```

Comment:
where do those numbers like `0.030927835051546393` come from? they should not be hardcoded directly in the function

Reply:
the numbers were calculated in the notebook based on the real data, 
move to const array.